### ISSUE TITLE
Issues sending messages

### ISSUE THREAD

**Issue by carusone4** [brismuth/google-voice-bulk-texter#126]
**Describe the bug**
I have been using this extension heavily for the past year, it is great. Recently I downloaded the new version of it about a week ago. Now when I send messages in google voice the first message I send doesn't come up and all I see is a buffering spinner in google voice where the message is supposed to be. Any ideas what this could be?

**URL**
The URL you were on when the problem occurred. 

**Expected behavior**

**Screenshots**

**Additional context**

**Comment by educatoral** [comment]
I keep getting the following error when send messages to multiple cell phone numbers for school, only the first one sends: 
Google Voice bulk texter v1.3.2:
Text failed. You can find support resources by opening the Google Voice Bulk Texter popup and clicking "Get Help" at the bottom.

Error: "confirmSent" failed. 

When you click "ok" the page will refresh.

**Comment by brismuth** [comment]
Is this still happening for you? Are you getting any error messages?

**Comment by safraw** [comment]
I am having issues with being blocked from sending messages; I am using the format i use to get people jobs . Would there be a way to prevent being blocked or seen as spam?

**Comment by Satie382x** [comment]
I am being blocked from sending messages because Google is tracking that I'm violating their terms of use.... i.e. sending multiple messages with the same content.

**Comment by brismuth** [comment]
Getting blocked by Google is a separate issue than the one reported here, and is unfortunately completely out of my control. See the support page:

https://github.com/brismuth/google-voice-bulk-texter/blob/master/support.md#account-suspension--error-confirmsent-failed
